flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/UnsignedTypeConversionITCase.java

```diff
@@ -95,7 +95,7 @@ public static void prepareMariaDB() throws IllegalStateException {
                     }
                 }
             } catch (Exception e) {
-                logger.warn("Initialize DB fail caused by {}", e);
+                logger.warn("Initialize DB fail caused by ", e);
```

Comment:
this should work properly?

Reply:
Sure, but the `{}` is an useless character.